e2e/testcases/helm_sync_test.go

```diff
@@ -211,39 +153,13 @@ func TestPublicHelm(t *testing.T) {
 func TestHelmWatchConfigMap(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.SyncSource, ntopts.Unstructured)
 
-	rs := k8sobjects.RootSyncObjectV1Beta1(configsync.RootSyncName)
-	nt.MustMergePatch(rs, `{
-		"spec": {
-		  "sourceFormat": "unstructured",
-		  "sourceType": "helm",
-		  "helm": {
-			"releaseName": "my-wordpress",
-			"namespace": "wordpress",
-			"auth": "none",
-			"repo": "https://charts.bitnami.com/bitnami",
-			"chart": "wordpress",
-			"version": "15.2.35",
-			"values": {
-			  "extraEnvVars": [
-				{
-				  "name": "TEST_1",
-				  "value": "val1"
-				},
-				{
-				  "name": "TEST_2",
-				  "value": "val2"
-				}
-			  ],
-			  "wordpressEmail": "test-user@example.com"
-			},
-			"valuesFileRefs": [
-			  {
-				"name": "helm-watch-config-map"
-			  }
-			]
-		  }
-		}
-	  }`)
+	rs := rootSyncForWordpressHelmChart(nt, func(m map[string]interface{}) {
+		delete(m, "wordpressUsername") // omit username so it can be set with values file
```

Comment:
Doesn't the values file overwrite the username?

Reply:
`values` takes precedence over `valuesFileRefs` it seems, which is what this test was verifying previously. I'm just retaining the assertion here. See [this assertion](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testcases/helm_sync_test.go#L352)

Reply:
ACK